src/main/java/frc/robot/subsystems/swerve/swervestatehelpers/AimAssistUtils.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.swerve.swervestatehelpers;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.subsystems.swerve.SwerveConstants;
+import frc.robot.subsystems.swerve.SwerveMath;
+
+import java.util.function.Supplier;
+
+import static frc.robot.subsystems.swerve.SwerveMath.getDriveMagnitude;
+
+public class AimAssistUtils {
+
+    public static ChassisSpeeds getRotationAssistedSpeeds(
+            ChassisSpeeds speeds,
+            ChassisSpeeds currentSpeeds,
+            Supplier<Rotation2d> robotRotationSupplier,
+            Supplier<Rotation2d> targetRotationSupplier,
+            SwerveConstants swerveConstants) {
```

Comment:
```suggestion
            SwerveConstants swerveConstants
            ) {
```
and remove space after (in all relevant places)